axelor-account/src/main/java/com/axelor/apps/account/web/AnalyticDistributionLineController.java

```diff
@@ -117,4 +127,60 @@ public void setAnalyticAxisDomain(ActionRequest request, ActionResponse response
               .getAnalyticAxisDomain(analyticMoveLine, company));
     }
   }
+
+  public void setAnalyticAccountDomain(ActionRequest request, ActionResponse response) {
+    String domain = "null";
+    List<Long> analyticAccountIdList = new ArrayList<>();
+    AnalyticDistributionLine analyticDistributionLine =
+        request.getContext().asType(AnalyticDistributionLine.class);
+    Context parentContext = request.getContext().getParent();
+    Context grandParentContext = null;
+
+    if (parentContext != null) {
```

Comment:
Check here if the parentContext class is AnalyticDistributionTemplate.class

Reply:
Added